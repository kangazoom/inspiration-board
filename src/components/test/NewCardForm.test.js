import React from 'react';
import NewCardForm from '../NewCardForm';
import { shallow } from 'enzyme';

describe('NewCardForm', () => {
  it('will match the NewCardForm snapshot', () => {

    const wrapper = shallow(
      <NewCardForm/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
