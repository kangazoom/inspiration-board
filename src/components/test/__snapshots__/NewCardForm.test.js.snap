// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCardForm will match the NewCardForm snapshot 1`] = `
<div>
  hi
</div>
`;
